src/neuroglancer/ui/layer_data_sources_tab.ts

```diff
@@ -362,6 +362,7 @@ export class LayerDataSourcesTab extends Tab {
       layerTypeDetection.classList.add('neuroglancer-layer-data-sources-tab-type-detection');
       layerTypeDetection.addEventListener('click', () => {
         changeLayerTypeToDetected(layer);
+        layer.managedLayer.layer!.toolBinder.loadDefaultKeybinds();
```

Comment:
Should probably be done in `changeLayerTypeToDetected` rather than here, since `changeLayerTypeToDetected` is also called when you press enter again with the URL input area focused